pkgs/development/compilers/rust/cargo-auditable-cargo-wrapper.nix

```diff
@@ -1,10 +1,13 @@
-{ lib, writeShellScriptBin, cargo, cargo-auditable }:
+{ lib, runCommand, makeBinaryWrapper, cargo, cargo-auditable }:
 
-(writeShellScriptBin "cargo" ''
-  export PATH="${lib.makeBinPath [ cargo cargo-auditable ]}:$PATH"
-  CARGO_AUDITABLE_IGNORE_UNSUPPORTED=1 exec cargo auditable "$@"
-'') // {
+runCommand "auditable-${cargo.name}" {
+  nativeBuildInputs = [ makeBinaryWrapper ];
   meta = cargo-auditable.meta // {
     mainProgram = "cargo";
   };
-}
+} ''
+  mkdir -p $out/bin
+  makeWrapper ${cargo}/bin/cargo $out/bin/cargo \
+      --set CARGO_AUDITABLE_IGNORE_UNSUPPORTED 1 \
+      --prefix PATH : ${lib.makeBinPath [ cargo cargo-auditable ]}
```

Comment:
```suggestion
    --set CARGO_AUDITABLE_IGNORE_UNSUPPORTED 1 \
    --prefix PATH : ${lib.makeBinPath [ cargo cargo-auditable ]}
```

Reply:
>  Use 2 spaces of indentation per indentation level in Nix expressions, 4 spaces in shell scripts. 

I've always read this is meaning 4 spaces inside shell scripts inside Nix expressions, but I suppose the other reading is valid.